arm-operationalinsights/2015-03-20/swagger/OperationalInsights.json

```diff
@@ -0,0 +1,1146 @@
+            "schema": {
+              "$ref": "#/definitions/StorageInsight"
+            },
+            "description": "The parameters required to create or update a storage insight."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created response definition.",
+            "schema": {
+              "$ref": "#/definitions/StorageInsight"
+            }
+          },
+          "200": {
+            "description": "OK response definition.",
+            "schema": {
+              "$ref": "#/definitions/StorageInsight"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.OperationalInsights/workspaces/{workspaceName}/storageInsightConfigs/{storageInsightName}": {
+      "get": {
+        "tags": [
+          "StorageInsights"
+        ],
+        "operationId": "StorageInsights_Get",
+        "description": "Gets a storage insight instance.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource group name of the storage insight."
+          },
+          {
+            "name": "workspaceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the workspace that contains the storage insight."
+          },
+          {
+            "name": "storageInsightName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the storage insight"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK response definition.",
+            "schema": {
+              "$ref": "#/definitions/StorageInsight"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "StorageInsights"
+        ],
+        "operationId": "StorageInsights_Delete",
+        "description": "Deletes a storage insight instance.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The resource group name of the storage insight."
+          },
+          {
+            "name": "workspaceName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the workspace that contains the storage insight."
```

Comment:
Log Analytics Workspace name that will contain the storageInsightsConfigs resource